dev-docs/examples/multi-format-example.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I am seeing "Prebid.js Multi-Format Example" appear in the left nav under "Developer Quick Start" on all /dev-docs/ pages.  Is this intended?

Reply:
Nope!  Forgot to add a `hide` key in the YAML header.  Addressed in d2df4d0